packages/core/template/default/index.md

```diff
@@ -1,132 +1,75 @@
 <frontmatter>
-  title: Landing Page
+  title: Home Page
   layout: default.md
-  pageNav: 2
-  pageNavTitle: "Chapters of This Page"
+  pageNav: 4
+  pageNavTitle: "Topics"
 </frontmatter>
 
 <br>
 
-<div class="bg-primary text-white py-5 mb-4">
+<div class="bg-primary text-white px-2 py-5 mb-4">
   <div class="container">
-    <h1 class="display-4 no-index">Landing Page Title</h1>
-    <p class="lead">A tagline can go here</p>
+    <h1 class="display-5 no-index">Great!<br>You've just initialized a MarkBind site.</h1>
+    <p class="lead">Let's get started...</p>
   </div>
 </div>
 
-# Heading 1
-Some text some text some text some text some text some text some text. **Some text some text some text some text some text ==some text== some text**. Some text some text some text some text some text some text some text some text some text some text some text some text some text some text. Some text some text some text some text some text some text. Some text some text some text some text some text some text some text.
+---
 
-**A block quote:**
+## What did you just do?
 
-> Some text some text some text some text some text some text some text. Some text some text some text some text some text some text some text. Some text some text some text some text some text some text some text some text some text some text some text some text some text some text. Some text some text some text some text some text some text. Some text some text some text some text some text some text some text.
+You have just initialized a _default_ MarkBind site! It is equipped with a set of core features, including site and page navigation. Additionally, we have included some convenient links to our User Guide, to help you get started quickly and easily.
 
-**A list:**
+<box type="tip">
 
-* item A
-* item B
-  1. item b1
-  1. item b2
-  1. item b3
+If you were intending to convert an existing GitHub wiki or a docs folder into MarkBind, use the "--convert" flag instead. See <a href="https://markbind.org/userGuide/markBindInTheProjectWorkflow.html#converting-existing-project-documentation-wiki" target="_blank">User Guide: MarkBind in the Project Workflow</a> for more information.
```

Comment:
```suggestion
If you were intending to convert an existing GitHub wiki or a docs folder into MarkBind, use the `--convert` flag instead. See <a href="https://markbind.org/userGuide/markBindInTheProjectWorkflow.html#converting-existing-project-documentation-wiki" target="_blank">User Guide: MarkBind in the Project Workflow</a> for more information.
```
Might be better to use the inline code format? Same for the --template below

Reply:
Thanks for the suggestion! Updated in [eface18](https://github.com/MarkBind/markbind/pull/2225/commits/eface184bfc22fc7a9db7f55e82882f47f018d5e).